ext/standard/mail.c

```diff
@@ -281,6 +320,11 @@ PHPAPI int php_mail(char *to, char *subject, char *message, char *headers, char
 		efree(f);
 	}
 
+	if (hdr && php_mail_detect_multiple_crlf(hdr)) {
+		php_error_docref(NULL TSRMLS_CC, E_WARNING, "Multiple or malformed newlines found in additional_header");
+		MAIL_RET(0);
```

Comment:
https://tools.ietf.org/html/rfc2822#section-2.2.3

According to RFC2822, you cannot send multiple CRLF, but CRLF+'Space'+CRLF.
Therefore, I presume the code is broken. Please let me know if I missed something.